dispatches/models/fossil_case/ultra_supercritical_plant/ultra_supercritical_powerplant.py

```diff
@@ -0,0 +1,1424 @@
+            "property_package": m.fs.prop_water,
+            "has_pressure_change": True
+        }
+    )
+
+    ###########################################################################
+    #  Add Condenser Mixer, Condenser, and Condensate pump                    #
+    ###########################################################################
+    # condenser mix
+    m.fs.condenser_mix = HelmMixer(
+        default={
+            "momentum_mixing_type": MomentumMixingType.none,
+            "inlet_list": ["main", "bfpt", "drain", "makeup"],
+            "property_package": m.fs.prop_water,
+        }
+    )
+
+    # Condenser is set up as a heater block
+    m.fs.condenser = Heater(
+        default={
+            "dynamic": False,
+            "property_package": m.fs.prop_water,
+            "has_pressure_change": False
+        }
+    )
+
+    # condensate pump
+    m.fs.cond_pump = HelmIsentropicCompressor(
+        default={
+            "property_package": m.fs.prop_water,
+        }
+    )
+    ###########################################################################
+    #  Feedwater heater declaration                                     #
+    ###########################################################################
+    # Feed water heaters (FWHs) are declared as 0D heat exchangers
+    # Shell side (side 1) is for steam condensing
+    # Tube side (side 2) is for feed water heating 
+
+    # Declaring indexed units for feed water heater mixers
+    # The indices reflect the connection to corresponding feed water heaters
+    # e. g. the outlet of fwh_mixer[1] is connected to fwh[1]
+    # Note that there are no mixers before FWHs 5 and 9
+    m.mixer_list = [1, 2, 3, 4, 6, 7, 8]
+    m.fs.fwh_mixer = HelmMixer(
+        m.mixer_list,
+        default={
+            "momentum_mixing_type": MomentumMixingType.none,
+            "inlet_list": ["steam", "drain"],
+            "property_package": m.fs.prop_water,
+        }
+    )
+
+
+    # Declaring indexed units for feed water heaters
+    m.fs.fwh = HeatExchanger(
+        pyo.RangeSet(9),
+        default={
+            "delta_temperature_callback": delta_temperature_underwood_callback,
+            "shell": {
+                "property_package": m.fs.prop_water,
+                "material_balance_type": MaterialBalanceType.componentTotal,
+                "has_pressure_change": True,
+            },
+            "tube": {
+                "property_package": m.fs.prop_water,
+                "material_balance_type": MaterialBalanceType.componentTotal,
+                "has_pressure_change": True,
+            },
+        }
+    )
+
+    ###########################################################################
+    #  Add deaerator                              #
+    ###########################################################################
+    m.fs.deaerator = HelmMixer(
+        default={
+            "momentum_mixing_type": MomentumMixingType.none,
```

Comment:
I assume you have a constraint elsewhere to set the outlet pressure in the deaerator?

Reply:
changed to `MomentumMixingType.minimize`